### PR TITLE
Merchant

### PR DESCRIPTION
**Pull request by jamogriff**
This PR implements the Merchant class with passing tests. The MerchantRepository  class is initially made, but needs additional things added to get the initial test to pass.

### CODE REVIEW

File: lib/merchant_repository.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+class MerchantRepository
+
+  def initialize(parsed_csv)
+    @merchants = parsed_csv
+
+    # turn parsed data into Ruby objects
+  end
```
Comment (lcpulzone) [inline lib/merchant_repository.rb:8]: I think this is a great idea.

File: lib/sales_engine.rb [hunk 2]
```diff
@@ -28,7 +28,7 @@ def self.parse_csv(path)
   end
 
   def merchants
-    MerchantRepository.new
+    MerchantRepository.new(parse_csv(@merchants))
```
Comment (lcpulzone) [inline lib/sales_engine.rb:31]: Into this!

**Review by lcpulzone — COMMENTED** [review]
Think this is a great start, stoked about how much we got done today!

**Comment by aliyamerali** [comment]
Echoing Leigh, this looks great!

File: lib/merchant_repository.rb (hunk 1 above)
Comment (aliyamerali) [reply lib/merchant_repository.rb:8]: ++

**Review by aliyamerali — APPROVED** [review]
Looking good!

**Comment by matthewjholmes** [comment]
Thanks.

**Pull request merged**